packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -90,33 +97,70 @@ contract("PricelessPositionManager", function(accounts) {
     assert.notEqual(beneficiaryCollateralBalance.toString(), "0");
   };
 
-  before(async function() {
+  const proposeAndSettleOptimisticOraclePrice = async (priceFeedIdentifier, requestTime, price) => {
```

Comment:
This is the basic method that I used to replace most instances of `await mockOracle.pushPrice`.